node/src/chain_spec/dev.rs

```diff
@@ -75,7 +80,12 @@ pub fn dev_config() -> Result<BatteryStationChainSpec, String> {
                         super::battery_station::DEFAULT_STAKING_AMOUNT_BATTERY_STATION,
                     )],
                     collator_commission: DefaultCollatorCommission::get(),
-                    inflation_info: crate::chain_spec::DEFAULT_COLLATOR_INFLATION_INFO,
+                    inflation_info: inflation_config(
+                        Perbill::from_parts(20),
```

Comment:
Don't we need from_percent() here?

Reply:
We could use `from_percent` for the case of event percentage, such as the highlighted `2%`. However, the other parts of the configuration also use one decimal fraction place, like `3.5%`, that's why we need `from_parts`, it maps `x / 1000`, such as `35 / 1000 = 3.5%`. To keep it uniform, I just used `from_parts` for any value in the configuration.

Reply:
How the heck did I assume `from_parts` is `x / 1_000` for `Perbill`? :laughing: It's `x / 1_000_000_000`.

https://github.com/zeitgeistpm/zeitgeist/pull/960/commits/ee29b50d6e7a5098e6a6b05bbe3ce1587429e27d, thanks Vivek!